pkg/dbmate/mysql.go

```diff
@@ -239,6 +239,29 @@ func (drv MySQLDriver) DeleteMigration(db Transaction, version string) error {
 	return err
 }
 
+// Acquires a change lock by setting an
+// [exclusive lock](https://dev.mysql.com/doc/refman/5.7/en/locking-functions.html#function_get-lock). NB! Locks are
+// reference counted and hence you must call `ReleaseChangeLock` the same number of times you call `AcquireChangeLock`
+// successfully
```

Comment:
what happens if the dbmate process crashes and doesn't release the lock?

Reply:
Its released on session termination, but probably that should be clarified.